R/read_redcap.R

```diff
@@ -72,6 +72,14 @@ read_redcap <- function(redcap_uri,
                         forms = NULL,
                         export_survey_fields = TRUE,
                         suppress_redcapr_messages = TRUE) {
+
```

Comment:
Please take a look at https://github.com/OuhscBbmc/REDCapR/blob/main/R/redcap-read.R to see how @wibeasley is checking `redcap_read()`'s parameters. I don't fully understand the need for some of the parameters but would be good to check we're not missing anything.

Reply:
- `any.missing = FALSE` disallows `NA`s. Makes sense to me to add this
- `pattern="^.{1,}$"` disallows empty strings (`""`). I could go either way here on needing this
- I see Will has `any.missing = TRUE` for `forms`. Not sure we want to accepts `forms` inputs with `NA`s